### PR TITLE
Fix GIVstream history table position

### PR DESCRIPTION
**Pull request by pkretzschmar**
Ref #388

### CODE REVIEW

File: src/components/views/userPublicProfile/ProjectsTable.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const ProjectsTable: FC<IProjectsTable> = ({
 			title = 'Listed';
 		} else if (listed === null) {
 			color = semanticColors.blueSky;
-			title = 'Waiting for review';
+			title = 'Draft';
```
Comment (RamRamez) [inline src/components/views/userPublicProfile/ProjectsTable.tsx:69]: When working on an issue, please create new branch from develop.

File: src/components/homeTabs/GIVstream.tsx [hunk 2]
```diff
@@ -238,6 +237,7 @@ export const TabGIVstreamBottom = () => {
 						</GsDataBlock>
 					</Col>
 				</Row>
+				<GIVstreamHistory />
```
Comment (RamRamez) [inline src/components/homeTabs/GIVstream.tsx:240]: what about history title? shouldn't it be moved too?

**Review by RamRamez — CHANGES_REQUESTED** [review]

File: src/components/homeTabs/GIVstream.tsx (hunk 2 above)
Comment (pkretzschmar) [reply src/components/homeTabs/GIVstream.tsx:240]: Done!

File: src/components/views/userPublicProfile/ProjectsTable.tsx (hunk 1 above)
Comment (RamRamez) [inline src/components/views/userPublicProfile/ProjectsTable.tsx]: This part is still wrong!

**Review by RamRamez — APPROVED** [review]

**Pull request merged**